data_steward/cdr_cleaner/cleaning_rules/clean_survey_conduct_recurring_surveys.py

```diff
@@ -0,0 +1,176 @@
+"""
+Inserts data representing the version of a survey into survey_conduct table for surveys with custom ids. Ex COPE/Minute
```

Comment:
nit: I think `update` or `correct` is more accurate for this CR instead of `insert`.